src/Main.java

```diff
@@ -0,0 +1,29 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Missing functionality: require 'search' as first argument. 

Should main handle all code? Consider moving code to separate methods.